src/surfaces/filtersDrawer/filtersDrawerNestedTogglesValueLabel.jsx

```diff
@@ -25,8 +28,10 @@ class FiltersDrawerNestedTogglesValueLabel extends React.PureComponent {
     }
 
     render() {
-        const { option: { label } } = this.props;
+        const { nestedTogglesData, option: { label } } = this.props;
         const containerClasses = ClassNames('page_filters_drawer--nested_toggles_value_label');
+        const clearable = get(nestedTogglesData, 'clearable', true);
+        const canClear = clearable || size(nestedTogglesData.value) > 1;
```

Comment:
Really OR here?  Not AND?
i.e. not this?
```JavaScript
const canClear = clearable && size(nestedTogglesData.value) > 1;
```

Also, what if the size is only 1?  There's no circumstance under which that'd be clearable?  i.e. no use case for "nothing selected" versus "the one thing selected"?  I guess that'd be weird ... we shouldn't be using nested toggles in that case; we should be using a checkbox.  But just confirming the intent here, and whether we've thought through all conceivable use cases.

Reply:
@groberts314  `clearable` means here the ability to remove all options (like `Dropdown` same prop), so OR should be here.

Reply:
Just double-checking.  I'll take your word for it! :-)